src/data/breaking-things-data.ts

```diff
@@ -0,0 +1,56 @@
+import { BreakACDifficultyTypeGroup, BreakHPDifficultyTypeGroup } from '../types/dc-types';
+
+// data
+export const OBJECT_BREAK_HP_INFO: BreakHPDifficultyTypeGroup = [
+    {
+        text: 'Tiny (bottle, lock)',
+        fragile: '2 (1d4)',
+        resilient: '5 (2d4)',
+    },
+    {
+        text: 'Small (chest, lute)',
+        fragile: '3 (1d6)',
+        resilient: '10 (3d6)',
+    },
+    {
+        text: 'Medium (barrel, chandelier)',
+        fragile: '4 (1d8)',
+        resilient: '18 (4d8)',
+    },
+    {
+        text: 'Large (cart, 10-ft.-by-10-ft. window)',
+        fragile: '5 (1d10)',
+        resilient: '27 (5d10)',
+    },
+];
+
+export const OBJECT_BREAK_AC_INFO: BreakACDifficultyTypeGroup = [
+    {
+        text: 'Cloth, paper, rope',
+        ac: '11',
```

Comment:
I don't think these values need to be strings

Reply:
Are you talking about the AC values?

I was keeping them as strings to be consistent with the other Types that store DCs as strings.

Reply:
The AC values, yeah. Ultimately not changing them to staty was an error -- it's less about "this is what we did before" and more about "this makes sense for the type" if all Difficulty Checks are doing to have values that are numeric, we should treat them as such.

Reply:
addressed in 1ca2df8